.github/workflows/lint.yml

```diff
@@ -12,12 +12,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout Code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v3.1.2
         with:
           # Full git history is needed to get a proper list of changed files within `super-linter`
           fetch-depth: 0
       - name: Lint Code Base
         uses: github/super-linter@v4
         env:
-          VALIDATE_ALL_CODEBASE: false
+          VALIDATE_ALL_CODEBASE: True
```

Comment:
We don't want to make this change, as it would be quite disruptive for most contributions. We only want files involved a PR to be linted. If you would like to lint the codebase in it's entirety I suggest doing that on a local clone with `shellcheck`.

Reply:
ok.